packages/block-editor/src/store/index.js

```diff
@@ -32,14 +32,8 @@ export const storeConfig = {
  */
 export const store = createReduxStore( STORE_NAME, {
 	...storeConfig,
-	persist: [ 'preferences' ],
 } );
 
-// We will be able to use the `register` function once we switch
-// the "preferences" persistence to use the new preferences package.
-const registeredStore = registerStore( STORE_NAME, {
-	...storeConfig,
-	persist: [ 'preferences' ],
-} );
```

Comment:
This is why I'm interested in finishing this PR 🙂 I'd like to remove all usages of the deprecated `registerStore` function and the `persist` plugin, and this is the last existing instance.